docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  postgres:
+    image: postgres:12
```

Comment:
Taking into account that GCP uses v13 as default, and thinking in the future, wouldn it be best to use postgres:13?

Reply:
Our CI tests currently use 12, so I think good for it to be consistent.

Reply:
Our github actions (in newer release branches) use version 12, so this is consistent with that. Hopefully this shouldn't make much of a difference